.github/workflows/codeql-analysis.yml

```diff
@@ -13,6 +13,15 @@ jobs:
     - uses: actions/checkout@v2
       with:
         fetch-depth: 2
+    - uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: |
+          3.1.x
+          5.0.x
+          6.0.x
+        source-url: https://nuget.pkg.github.com/graphql-dotnet/index.json
+      env:
+        NUGET_AUTH_TOKEN: ${{secrets.GITHUB_TOKEN}}
```

Comment:
It's better to deal with codeql in a separate PR.

Reply:
```
dotnet tool install -g dotnet-format --version 6.0.243104 --add-source https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json
          dotnet format --no-restore --verify-no-changes -v diag --severity warn whitespace || (echo "Run 'dotnet format' to fix whitespace issues" && exit 1)
          dotnet format --no-restore --verify-no-changes -v diag --severity warn analyzers || (echo "Run 'dotnet format' to fix analyzers issues" && exit 1)
```

This seems to be the updated execution script. I can add this into the PR if needed

Reply:
Yes, `--verify-no-changes` is the right option now. Also it's worth to change messages  after `echo` to `Run 'dotnet format --no-restore --verify-no-changes -v diag --severity warn whitespace' to fix whitespace issues`. The same for analyzers.

Reply:
How about bumping the version for dotnet format to 7.0.255502 (the latest on https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json)? The new options mentioned above work as expected

Reply:
Note that 6.0.243104 is the last version that supports netcoreapp3.1